ui/webui/src/components/storage/MountPointMapping.jsx

```diff
@@ -207,19 +206,25 @@ const FormatColumn = ({ deviceData, handleRequestChange, idPrefix, request }) =>
 
     return (
         <Flex>
-            <Checkbox
-              id={idPrefix + "-checkbox"}
-              isChecked={request.reformat}
-              isDisabled={isRootMountPoint}
-              label={_("Reformat")}
-              onChange={checked => handleRequestChange(request["mount-point"], request["device-spec"], request["request-id"], checked)}
-            />
+            {!isRootMountPoint &&
+                <Switch
+                  id={idPrefix + "-switch"}
+                  isChecked={request.reformat}
+                  isDisabled={isRootMountPoint}
+                  defaultChecked={false}
+                  aria-label="Reformat"
+                  onChange={checked => handleRequestChange(request["mount-point"], request["device-spec"], request["request-id"], checked)}
+                />}
             {isRootMountPoint &&
-                <Popover
-                  bodyContent={_("The root partition is always re-formatted by the installer.")}
-                  showClose={false}>
-                    <HelpIcon />
-                </Popover>}
+                <Tooltip
+                  content={_("The root partition is always re-formatted by the installer.")}>
+                    <Switch
```

Comment:
Please avoid duplicating code. Create a variable called switch and assign the Switch component.